ew/static/poi.py

```diff
@@ -990,7 +990,7 @@
         role="NLAC U",
         permissions={'nlac-university': ['read', 'send', 'connect']},
         pvp=False,
-        vendors=['College'],
+        vendors=['College', "Gumball Machine"],
```

Comment:
we don't need to add a bunch of food vendors in new places. let's keep this in poudrin alley to complement the 7/11 instead